cloudinit/sources/DataSourceOVF.py

```diff
@@ -81,314 +42,19 @@ def _get_data(self):
-                    self._vmware_cust_conf,
-                )
-
-            if special_customization:
-                LOG.debug("Applying password customization")
-                pwdConfigurator = PasswordConfigurator()
-                adminpwd = self._vmware_cust_conf.admin_password
-                try:
-                    resetpwd = self._vmware_cust_conf.reset_password
-                    if adminpwd or resetpwd:
-                        pwdConfigurator.configure(
-                            adminpwd, resetpwd, self.distro
-                        )
-                    else:
-                        LOG.debug("Changing password is not needed")
-                except Exception as e:
-                    _raise_error_status(
-                        "Error applying Password Configuration",
-                        e,
-                        GuestCustEvent.GUESTCUST_EVENT_CUSTOMIZE_FAILED,
-                        vmwareImcConfigFilePath,
-                        self._vmware_cust_conf,
-                    )
-
-                if customscript:
-                    try:
-                        postcust = PostCustomScript(
-                            customscript, imcdirpath, ccScriptsDir
-                        )
-                        postcust.execute()
-                    except Exception as e:
-                        _raise_error_status(
-                            "Error executing post-customization script",
-                            e,
-                            GuestCustEvent.GUESTCUST_EVENT_CUSTOMIZE_FAILED,
-                            vmwareImcConfigFilePath,
-                            self._vmware_cust_conf,
-                        )
-
-            if product_marker:
-                try:
-                    setup_marker_files(
-                        product_marker,
-                        os.path.join(self.paths.cloud_dir, "data"),
-                    )
-                except Exception as e:
-                    _raise_error_status(
-                        "Error creating marker files",
-                        e,
-                        GuestCustEvent.GUESTCUST_EVENT_CUSTOMIZE_FAILED,
-                        vmwareImcConfigFilePath,
-                        self._vmware_cust_conf,
-                    )
-
-            self._vmware_cust_found = True
-            found.append("vmware-tools")
-
-            # TODO: Need to set the status to DONE only when the
-            # customization is done successfully.
-            util.del_dir(os.path.dirname(vmwareImcConfigFilePath))
-            enable_nics(self._vmware_nics_to_enable)
-            set_customization_status(
-                GuestCustStateEnum.GUESTCUST_STATE_DONE,
-                GuestCustErrorEnum.GUESTCUST_ERROR_SUCCESS,
-            )
-            set_gc_status(self._vmware_cust_conf, "Successful")
-
         else:
             np = [
                 ("com.vmware.guestInfo", transport_vmware_guestinfo),
```

Comment:
Thanks a lot for the review, @TheRealFalcon 

Yes, I've updated the _get_subplatform() in DataSourceOVF.py as you suggested.

While the transport_vmware_guestinfo() function is still in DataSourceOVF.py, since this transport supports OVF format only which is same with transport_iso9660() function, so I left it intentionally.